### ISSUE TITLE
Getting negative values for test_calibration(tau.forest)

### ISSUE THREAD

**Issue by YongzhWang** [grf-labs/grf#1405]
Hi,

I am trying to fit a casual forest but get approx. -1 estimates for mean.forest.prediction. I wonder if it is common to get negative values for test_calibration(tau.forest). Does this just signal that the prediction is terrible?

Best,
Andrew

**Comment by erikcs** [comment]
Yes. You might find the [RATE](https://grf-labs.github.io/grf/articles/rate.html) easier to interpret for this kind of exercise.
